ext-src/utils/Config.ts

```diff
@@ -30,6 +30,11 @@ const NEXUS_IQ_PUBLIC_APPLICATION_ID = NEXUS_IQ_BASE.concat(".", "applicationId"
 const NEXUS_IQ_USER_PASSWORD = NEXUS_IQ_BASE.concat(".", "userPassword");
 const NEXUS_IQ_STRICT_SSL = NEXUS_IQ_BASE.concat(".", "strictSSL");
 
+const OSS_INDEX_BASE = "ossindex";
+
+const OSS_INDEX_USERNAME = OSS_INDEX_BASE.concat(".", "username");
+const OSS_INDEX_TOKEN = OSS_INDEX_BASE.concat(".", "userToken");
```

Comment:
Since this uses a different file, it may not matter, but in case you wanted to follow the same yaml content standard we have used elsewhere, it is defined here: https://pkg.go.dev/github.com/sonatype-nexus-community/go-sona-types@v0.0.12/configuration#example-ConfMarshallOssi
```
ossi:
  Username: ossiUser@email.com
  Token: ossiToken
```

Reply:
This is a great idea.